frontend/src/components/maps/leaflet/Map.tsx

```diff
@@ -184,7 +184,6 @@ const Map: React.FC<MapProps> = ({
   const [triggerFilterChanged, setTriggerFilterChanged] = useState(true);
   const [activeBasemap, setActiveBasemap] = useState<BaseLayer | null>(null);
   const smallScreen = useMediaQuery({ maxWidth: 1800 });
-  const [mapWidth, setMapWidth] = useState(0);
```

Comment:
switching to this style of the resize detector allows me to remove this state var.